### PR TITLE
fix(dic): config:dump-reference

### PR DESCRIPTION
**Pull request by jvancoillie**
| Q                | A
| ---------------- | ---
| PHP version?     | 7.4
| Bundle version?  | 0.4.2
| Symfony version? | 5.6.2
| New feature?     | no
| Bug fix?         | yes
| Discussion?      | #82

### CODE REVIEW

File: src/DependencyInjection/SchedulerBundleExtension.php [hunk 1]
```diff
@@ -968,4 +968,9 @@ private function registerDataCollector(ContainerBuilder $container): void
             ])
         ;
     }
+
+    public function getConfiguration(array $config, ContainerBuilder $container)
```
Comment (Guikingone) [inline src/DependencyInjection/SchedulerBundleExtension.php:972]: Return typehint to add if the interface allows to.

Comment (Guikingone) [reply src/DependencyInjection/SchedulerBundleExtension.php:972]: BTW, could we add a test for this change?

Comment (jvancoillie) [reply src/DependencyInjection/SchedulerBundleExtension.php]: of course

Comment (jvancoillie) [reply src/DependencyInjection/SchedulerBundleExtension.php]: is the test enough?

Comment (Guikingone) [reply src/DependencyInjection/SchedulerBundleExtension.php]: Yes

File: src/DependencyInjection/SchedulerBundleExtension.php [hunk 2]
```diff
@@ -968,4 +969,9 @@ private function registerDataCollector(ContainerBuilder $container): void
             ])
         ;
     }
+
+    public function getConfiguration(array $config, ContainerBuilder $container): ConfigurationInterface
```
Comment (Guikingone) [inline src/DependencyInjection/SchedulerBundleExtension.php:973]: Consider adding `{@inheritdoc}`

Comment (jvancoillie) [reply src/DependencyInjection/SchedulerBundleExtension.php:973]: I will replace :configurationInterface by the PHPdoc you are right !

Comment (Guikingone) [reply src/DependencyInjection/SchedulerBundleExtension.php:973]: That's not the idea, the return typehint can stay but you need to add the phpdoc for `{@inheritdoc}` 🙂

**Comment by Guikingone** [comment]
Looks good to me, thanks @jvancoillie 🚀

**Pull request merged**